src/ServerAdministration/WindowsService/WheelMUDService.cs

```diff
@@ -8,60 +8,37 @@
 // </summary>
 //-----------------------------------------------------------------------------
 
+using Topshelf;
```

Comment:
Having usings sorted on checkin helps eliminate noise from further checkins and sometimes eliminates minor merge conflicts (resulting in the same using being added twice because it was once added in one place, while another dev added it elsewhere).  Indeed not a big issue (especially as compiling immediately finds such accidental duplication).

The reason StyleCop puts usings inside the namespace is reasoned here: http://stackoverflow.com/questions/125319/should-using-statements-be-inside-or-outside-the-namespace
Granted, that reason is _pretty weak_.  I don't agree that avoiding those obscure cases is strong enough reason to adopt such an uncommon pattern (outside MS StyleCopped teams).  But I do think letting StyleCop be the complete arbitrator has enough value, that I should comply with the rules that I don't like in addition to the majority.

Absolutely agree about the donating time thing.  You'll note I accepted the PR without complaint, I just raised that we should talk about it  :)  In the past, as other devs rolled in and out, I'd eventually clean up after them - basically the StyleCop strategy was "if you don't want to worry about it, don't worry about it...  We love all the help we can get.  Just don't be surprised when David comes through the new code later with another StyleCop compliance pass."